server/models.js

```diff
@@ -0,0 +1,20 @@
+const db = require('./data/db');
+const bookshelf = require('bookshelf')(db);
+
+const WishList = bookshelf.model('Wishlist', {
+  tableName: 'wishlist',
+  items() {
+    return this.hasMany(WishListItem);
+  },
+});
+
+const WishListItem = bookshelf.model('WishListItem', {
+  tableName: 'item',
+  wishlist() {
+    return this.belongsTo(WishList);
+  },
+});
+
+exports.WishList = WishList;
+exports.WishListItem = WishListItem;
+exports.Bookshelf = bookshelf;
```

Comment:
Since we're using Node on the server side, if we capitalized Bookshelf when assigning the variable name on line 2 here, we could use some destructuring to export everything with one super-clean line:
`module.exports = {WishList, WishListItem,Bookshelf};`

Also, I really appreciate the separation of concerns here, keeping the code for DB interaction apart from the server/app.js makes for nice, clean code, as well as being modular by its nature (if we needed to replace our DB system with a different one, we could replace this file without modifying others, as long as the new one exports variables that have the same names and methods, etc.

Reply:
I like the destructuring idea